.replit

```diff
@@ -1,2 +1,2 @@
 language = "rust"
-run = "bash -c \"[ -d ~/.cargo ] || bash <(curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs) -y --default-toolchain nightly; ~/.cargo/bin/cargo build; RUST_LOG=info target/debug/starter-snake-rust\""
+run = "cargo run"
```

Comment:
By reverse engineering the default Rust repl this would be a complete `.replit` (with language server commented because poor 0.5vCPU)

```suggestion:-1
run = ["cargo", "run"]
entrypoint = "src/main.rs"

[packager]
language = "rust"

[packager.features]
packageSearch = true

[languages.rust]
pattern = "**/*.rs"


# From my personal experience rust-analyzer take everything on free tier
# [languages.rust.languageServer]
# start = "rust-analyzer"

```